src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -3,18 +3,133 @@
 import React from "react";
 import { Meta, Story } from "@storybook/react";
 import { Tooltip as TooltipComponent, TooltipProps } from "./Tooltip";
+import { BrightHeaderStyle } from "./BrightHeader";
+import IconIcons from "@elements/Icon/Generated/IconIcons";
 
 export default {
     title: "Components/Tooltip",
     component: TooltipComponent,
     args: {
-        tooltip: "Tooltip Content (Reactnode)",
+        tooltip: "Your text here",
+        heading: "",
+    },
+    argTypes: {
+        brightHeader: {
+            options: ["None", "Information", "Warning", "Tip", "Note"],
+            mapping: {
+                None: null,
+                Information: BrightHeaderStyle.Information,
+                Warning: BrightHeaderStyle.Warning,
+                Tip: BrightHeaderStyle.Tip,
+                Note: BrightHeaderStyle.Note,
+            },
+            defaultValue: "None",
+            control: { type: "select" },
+        },
+        tooltipIcon: {
+            options: ["None", "Icon"],
+            mapping: {
+                None: null,
+                Icon: <IconIcons key="1" />,
+            },
+            defaultValue: "None",
+            control: { type: "select" },
+        },
+        headingIcon: {
+            options: ["None", "Icon"],
+            mapping: {
+                None: null,
+                Icon: <IconIcons key="1" />,
+            },
+            defaultValue: "None",
+            control: { type: "select" },
+        },
     },
 } as Meta;
 
-export const Tooltip: Story<TooltipProps> = (args: TooltipProps) => (
-    <div>
-        <TooltipComponent {...args} />
-        <p>Some other text.</p>
-    </div>
-);
+const loremIpsum = "Cupcake ipsum dolor sit amet ice cream.";
+
+export const TooltipTemplate: Story<TooltipProps> = (args: TooltipProps) => <TooltipComponent {...args} />;
+
+export const Default = TooltipTemplate.bind({});
+Default.args = {
+    tooltip: loremIpsum,
```

Comment:
You defined `tooltip` on the global level (L13) so you can remove all `tooltip` from the stories